pymeasure/instruments/fwbell/fwbell5080.py

```diff
@@ -39,7 +40,7 @@ class FWBell5080(Instrument):
         meter = FWBell5080('/dev/ttyUSB0')  # Connects over serial port /dev/ttyUSB0 (Linux)
 
         meter.units = 'gauss'               # Sets the measurement units to Gauss
-        meter.range = 3e3                   # Sets the range to 3 kG
+        meter.range = 1                     # Sets the range to 3 kG
```

Comment:
Someway related to https://github.com/pymeasure/pymeasure/issues/666
It could be nice to use instrument with fixed unit e.g. Gauss and allow users to specify range with unit (using Pint) with software doing the conversion to instrument units behind the scenes.
Just a comment for possible future evolutions.

Reply:
I haven't tried Pint out yet, but read about it when it was added with https://github.com/pymeasure/pymeasure/pull/642

Part of move to [0,1,2] for range was to move away from `truncated_discrete_set` where it would accept whatever value but may send a different value to the instrument, which is not obvious to the user. It can also pollute the results file with parameters that weren't actually used by the instrument. That is a tangent but maybe a topic for Discussions.